### DISCUSSION TITLE
Failed to Send Password Recovery Email - Error Message Help Needed

### DISCUSSION THREAD

**Discussion by nocodedevteam3** [supabase/supabase#30493] [Questions]
Hi Supabase Community,

I'm having trouble with the password recovery feature in my application. When I try to send a password recovery email, I receive the following error:

Error message:
Failed to send password recovery: failed to make recovery request: Error sending recovery email.

I've attached a screenshot of the error for reference.

Does anyone know what might be causing this issue or how I can fix it? I'm not sure if it's related to email settings or some other configuration. Any help or pointers would be greatly appreciated!
![Screenshot 2024-11-15 175458](https://github.com/user-attachments/assets/70899ef3-f651-4c03-96c0-4ac073d32764)

**Comment by GaryAustin1** [comment]
You should look in the Auth logs of the dashboard to see if there is more info on what error is occurring.  
Are other types of emails being sent?

**Reply by GaryAustin1** [reply]
I asked if other types of emails (signup/signin) are being sent correctly.  You said No, but it was then unclear based on the rest of your response if they are working.  You should try another type of email to see if it is a general problem or just a reset email problem.

Your error seems to say it is a general problem and you don't have your SMTP setup correctly.
`Error: SMTP Error 535 SMTP error 535 indicates an issue with the client's authentication with the email server. It may be the result of many issues, some of the most common causes being: Invalid credentials. Disabled account.`

**Comment by nocodedevteam3** [comment]
![image](https://github.com/user-attachments/assets/03107552-1453-4324-a5a7-80713691a5ff)
no, I try different email, but not sent any password recovery email... I will attach error message log.
